hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -1816,8 +1810,267 @@
       }
     ],
 
-    // these CSRs are defined in a separate hjson specific to the closed source OTP wrapper.
+    // OTP wrapper-specific registers
```

Comment:
@arnonsha @zi-v @sha-ron 

This is how we intend the prim CSR mechanism to look like.
The structure is exposed in genericzized format (without descriptions and names) so that we can generate matching DV RAL model and SW masks.
On the closed-source side we will be able to reuse the existing Hjson (with few modifications) to create an alias layer inside the regtool.
Overall, modifications to the closed-source code are being kept to a minimum with this approach (see Gerrit CL).
This basically